src/CalcViewModel/StandardCalculatorViewModel.cpp

```diff
@@ -222,10 +224,32 @@ void StandardCalculatorViewModel::SetParenthesisCount(_In_ const wstring& parent
     if (IsProgrammer || IsScientific)
     {
         OpenParenthesisCount = ref new String(parenthesisCount.c_str());
-        RaisePropertyChanged("LeftParenthesisAutomationName");
     }
 }
 
+void StandardCalculatorViewModel::OpenParenthesisCountNarratorAnnouncment()
```

Comment:
The phrasing of this function name is somewhat ambiguous.  Probably, this should be `Set...` so it's clear that this is setting an announcement.